packages/block-library/src/query/constants.js

```diff
@@ -1,7 +1,9 @@
 export const MAX_FETCHED_TERMS = 100;
+export const MAX_SUGGESTED_POSTS = 100;
```

Comment:
Since we are fetching suggestions, it would be probably better to have a lower limit like 20 or something.